scaleway/resource_rdb_user.go

```diff
@@ -70,7 +70,7 @@ func resourceScalewayRdbUserCreate(ctx context.Context, d *schema.ResourceData,
 
 	ins, err := waitInstance(ctx, rdbAPI, region, instanceID)
 	if err != nil {
-		return diag.FromErr(err)
+		diag.FromErr(err)
```

Comment:
Why is this change needed?

Reply:
oops copy/paste error. I've juste added the return